invalid_settings.yaml.enc

```diff
@@ -1 +1 @@
-6YOGfLbteljFpkqxM3Ylv8mmumsEIcU1dKdYOKQ9YlSIrgMup384CZ7FEJQXdZvjMJYd59UC5OqqWzZMLOTAolNa/jxfpFjMg7ewzIbTdIYEezeY8mNcAw0rY4PQQ/FBr/LWsBsQBlUqSqsPeS/2Gpm3fCfJv9xrYypdUqGtA8si7ZdXc3W5L2TCBVWBINEQ--AJlY4OKiXsBawPXt--h55JI+mmBZI/huaiCTjVsQ==
\ No newline at end of file
+CQs//AJr6T3ChK1Q0y7NAHgIYU13JwcyhWnVIKU2D7SINUieiuKYFyNur7sp6tj3RXFEWCbvnm2ljgFnYl0BPAQnGEJnEEn550WkdyuP4biTDMtFUcQWq2Ps7tnjjEDG9zZw7B3qvNXa+/2ZtvrehScjSMnRV53h7Q93Q6TDPv+WBpRIMk6GV8UU--EiRdtykeBb2CCQWv--OHViftm9O1NqCG7aU+SJ9w==
```

Comment:
what is the purpose of changing the invalid settings content?

Reply:
A change was made in the connection options. The connector supported custom domain names, which has bene removed in the latest version as it is not applicable anymore. The spec was a custom domain.